### PR TITLE
Release 5.1.0

### PR DESCRIPTION
**Pull request by johnhwhite**
- Added the lookup column type. [#157](https://github.com/blackbaud/skyux-ag-grid/pull/157)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 5.1.0 (2021-10-28)
+
+- Added the lookup column type. [#157](https://github.com/blackbaud/skyux-ag-grid/pull/157)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: Should `column` be `cell` like in the PR title?

Comment (johnhwhite) [reply CHANGELOG.md]: Updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 5.1.0 (2021-10-28)
```
Comment (blackbaud-paulah) [inline CHANGELOG.md:1]:
```suggestion
# 5.1.0 (2021-10-29)
```

**Review by blackbaud-johnly — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (blackbaud-johnly) [reply CHANGELOG.md:1]: D'oh! Nice catch. My bad, @johnhwhite.

**Review by blackbaud-paulah — APPROVED** [review]

**Pull request merged**